### ISSUE TITLE
Include refactored loss in training script using metrics

### ISSUE THREAD

**Issue by oarriaga** [oarriaga/paz#16]
1. Make unit tests for new loss function: localization, negative classification and positive classification
2. Refactor training script such that is uses new loss
3. Refactor training script such that is uses as metrics the localization, negative classification and positive classification functions.
